bin/test_unit

```diff
@@ -36,3 +39,11 @@ docker run --rm \
   bash -exc "
     cat ./junit.output | go-junit-report > ./junit.xml
   "
+
+# format coverage output XML
+docker run --rm \
+  -v $PWD/:/test \
+  secrets-provider-for-k8s-junit:latest \
+  bash -exc "
+    gocov convert ./c.out | gocov-xml > ./coverage.xml
+  "
```

Comment:
can't we combine these two? So the already existing container will run both commands instead of creating 2 of them?